sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -249,36 +251,52 @@ export class MessageSender extends LinkEntity {
    * @param sendBatch Boolean indicating whether the encoded message represents a batch of messages or not
    * @return {Promise<Delivery>} Promise<Delivery>
    */
-  private _trySend(encodedMessage: Buffer, sendBatch?: boolean): Promise<void> {
-    const retryOptions = this._retryOptions || {};
-    retryOptions.timeoutInMs = getRetryAttemptTimeoutInMs(retryOptions);
+  private _trySend(
+    encodedMessage: Buffer,
+    sendBatch: boolean,
+    options: OperationOptions | undefined
+  ): Promise<void> {
+    const abortSignal = options?.abortSignal;
+
     const sendEventPromise = () =>
       new Promise<void>(async (resolve, reject) => {
-        const actionAfterTimeout = () => {
-          const desc: string =
-            `[${this._context.namespace.connectionId}] Sender "${this.name}" ` +
-            `with address "${this.address}", was not able to send the message right now, due ` +
-            `to operation timeout.`;
-          log.error(desc);
-          const e: AmqpError = {
-            condition: ErrorNameConditionMapper.ServiceUnavailableError,
-            description: desc
-          };
-          return reject(translate(e));
-        };
+        let initTimeoutTimer: any;
```

Comment:
@chradek - meant to ask you this before. TS has this typed as a NodeJS.Timer (which I assume wouldn't work if I was trying to use this in the browser). What do we declare at the type in that case?

Reply:
Ah, never mind. I found another spot where we just _do_ type it with a NodeJS.Timer (`batchingReceiver.ts`, line 87).

I can add that in but I'm still interested if there's a better technique (or if this is, as apparently we have in batchingReceiver, not an issue)

Reply:
Honestly, I think `any` is probably fine, but as long as it isn't exported from our package it shouldn't really matter if we type it as a NodeJS.Timer or a number (non-node.js environments).

Reply:
Okay, I'll leave it as is for now. The not-exported part was the important part, it appears.